pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```diff
@@ -98,6 +98,26 @@ async def run(self):
         for arch, digest in digest_by_arch.items():
             await self.sync_to_mirror(arch, bootc_build.el_target, f"{repo_url}@{digest}")
 
+        if not self.runtime.dry_run:
+            if bootc_build.embargoed:
+                await self.sync_to_quay(bootc_build.image_pullspec, ART_PROD_PRIV_IMAGE_REPO)
+            else:
+                await self.sync_to_quay(bootc_build.image_pullspec, ART_PROD_IMAGE_REPO)
+        else:
+            self._logger("Skipping sync to quay.io/openshift-release-dev/ocp-v4.0-art-dev since in dry-run mode")
```

Comment:
Imo this can move to the `self.sync_to_quay` func

Reply:
True - but I was wondering if we need to enter the function at all, sync build won't exist in dry-run mode

Reply:
Hmm right, things would also break early with dry-run looking above this piece of code. Anything that depends on the build would fail. Okay this should be fine